src/ripple/beast/core/LexicalCast.h

```diff
@@ -126,7 +149,8 @@ struct LexicalCast<Out, char const*>
     bool
     operator()(Out& out, char const* in) const
     {
-        return LexicalCast<Out, std::string>()(out, in);
+        assert(in != 0);
```

Comment:
I think it would be more conventional to write this as `assert(in != nullptr)` or (my preference) `assert(in)`.

Reply:
fixed in https://github.com/XRPLF/rippled/commit/85e001dcb21516ad0d711e60af70fc9141f29162